src/js/_enqueues/admin/site-icon.js

```diff
@@ -1,138 +1,180 @@
-(function($) {
+/* global wp, jQuery */
+
+/**
+ * Calculates image selection options based on the attachment dimensions.
+ *
+ * @since 6.5.0
+ *
+ * @param {Object} attachment The attachment object representing the image.
+ * @return {Object} The image selection options.
+ */
+function calculateImageSelectOptions( attachment ) {
```

Comment:
I'm not sure moving `calculateImageSelectOptions` and `switchToUpdate` out from the Immediately Invoked Function Expression (IIFE) is recommended. An IIFE also serves the purpose of 'encapsulating' other functions and variables in the IIFE function scope. Instead, to my understanding, these functions would be now in the global scope, which is not recommended. I think most of the WordPress JS files use an IIFE  for this reason: avoid global scope pollution and potential names collision.

Reply:
I guess this is resolved with [edb1546](https://github.com/WordPress/wordpress-develop/pull/6109/commits/edb154606304840d57181c6c9c965eee68541dc1)